tests/common/snappi_tests/snappi_fixtures.py

```diff
@@ -849,8 +850,9 @@ def get_multidut_tgen_peer_port_set(line_card_choice, ports, config_set, number_
     elif line_card_choice in ['chassis_multi_line_card_multi_asic']:
         # Different line card and minimum one port from different asic number
         if len(linecards.keys()) >= 2:
-            host_asic = list(zip(config_set[line_card_choice]['hostname'], config_set[line_card_choice]['asic']))
-            peer_ports = list(zip_longest(*[linecards[host][asic] for host, asic in host_asic]))
+            host_asic = list(product(config_set[line_card_choice]['hostname'], config_set[line_card_choice]['asic']))
```

Comment:
If  one LC has single asic, another LC has 3 asics. Will this logic work?
In 'asic', only mention the asic being used for testing.

Reply:
@selldinesh to  check the single asic and 3 asic combination logic, and open PR if needed.